crawl-ref/source/mon-cast.h

```diff
@@ -57,6 +57,7 @@ spell_type living_spell_type_for(monster_type mtyp);
 
 monster* cast_phantom_mirror(monster* mons, monster* targ,
                              int hp_perc = 35,
-                             int summ_type = SPELL_PHANTOM_MIRROR);
+                             int summ_type = SPELL_PHANTOM_MIRROR,
+                             int swap_chance_reciprocal = 2);
```

Comment:
`swap_chance_reciprocal` is pretty verbose, and requires the caller to have big opinions on what's going on inside the function. Passing in which summon this is (first? second?) is a nicer API to explain.